dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -153,68 +149,99 @@ private Response postWithEndpointState(String endpointId, String url, MediaType
 		return response;
 	}
 
+    /**
+     * Resolves remote IP address from request.
+     * @param request {@link HttpServletRequest}
+     * @return a String representing the remote IP address (or hostname)
+     */
+    private static String resolveRemoteIp(@Context final HttpServletRequest request) {
+        final String remoteIP = request.getRemoteHost();
+        return !UtilMethods.isSet(remoteIP) ? remoteIP : request.getRemoteAddr();
+    }
+
+    private static String getFullLocalIp(@Context final HttpServletRequest request) {
+        final String localIp = request.getLocalName();
+        Optional<String> port = HttpRequestDataUtil.getServerPort();
+        return (!UtilMethods.isSet(localIp) ? localIp : request.getLocalName())
+                + ':' + port.orElse(request.getLocalPort() + "");
```

Comment:
Codacy found an issue: [Do not add empty strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)